.code-samples.meilisearch.yaml

```diff
@@ -41,21 +41,21 @@ faceted_search_2: |-
       IndexSearchQuery(
         indexUid: 'books',
         facets: ['language', 'genres', 'author', 'format'],
-        filter: [['language = English', 'language = French'], ['genres = Fiction']]
+        filter: '(language = English AND language = French) OR genres = Fiction'
```

Comment:
@brunoocasali what do you think about using `filterExpression` here ?

Reply:
Yes, I think it is time to share the solution with the public, feel free to refactor some of the code-samples :D